Makefiles don’t work on macOS

**Describe the bug**
Makefiles fail to work on macOS after #107. A libsdl2 error is raised.

**To Reproduce**
Steps to reproduce the behaviour:
0. Try this on macOS Sonoma 
1. Compile any code using Makefiles like `4_cv_basics/1_image_representation/Makefile`.
2. Get `main.cpp:28:10: fatal error: 'SDL2/SDL.h' file not found`

**Expected behavior**
The file should have compiled successfully to give an executable.

**Screenshots**
<img width="682" alt="image" src="https://github.com/SRA-VJTI/Pixels_Seminar/assets/40035606/212b7e50-68f1-4c1d-b7fd-597ff1c0e952">


**Desktop (please complete the following information):**
 - OS: macOS

--- a/4_cv_basics/1_image_representation/Makefile
+++ b/4_cv_basics/1_image_representation/Makefile
@@ -5,9 +5,9 @@ CC = g++
 PROJECT = image_representation		
 
 # Here Include specifies which directories can be added in the include path of the main.cpp
-CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) -Iinclude/		
+CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) $(shell pkg-config --cflags sdl2)		
 # What libraries to be imported here it is opencv and SDL
-LIBS = $(shell pkg-config --libs opencv4) -lSDL2
+LIBS = $(shell pkg-config --libs opencv4) $(shell pkg-config --libs sdl2)
 
 # Similar to how we used to link ad make object files in the terminal
 # the command make build SRC=<FILENAME> should build the executable in the subfolder

--- a/4_cv_basics/2_playing_with_images/Makefile
+++ b/4_cv_basics/2_playing_with_images/Makefile
@@ -5,9 +5,9 @@ CC = g++
 PROJECT = playing_with_images		
 
 # Here Include specifies which directories can be added in the include path of the main.cpp
-CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) $(shell pkg-config --cflags sdl2)		
+CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) -Iinclude/		
 # What libraries to be imported here it is opencv
-LIBS = $(shell pkg-config --libs opencv4) $(shell pkg-config --libs sdl2)
+LIBS = $(shell pkg-config --libs opencv4)
 
 # Similar to how we used to link ad make object files in the terminal
 # the command make build SRC=<FILENAME> should build the executable in the subfolder

--- a/4_cv_basics/2_playing_with_images/Makefile
+++ b/4_cv_basics/2_playing_with_images/Makefile
@@ -5,9 +5,9 @@ CC = g++
 PROJECT = playing_with_images		
 
 # Here Include specifies which directories can be added in the include path of the main.cpp
-CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) -Iinclude/		
+CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) $(shell pkg-config --cflags sdl2)		
 # What libraries to be imported here it is opencv
-LIBS = $(shell pkg-config --libs opencv4) 
+LIBS = $(shell pkg-config --libs opencv4) $(shell pkg-config --libs sdl2)
 
 # Similar to how we used to link ad make object files in the terminal
 # the command make build SRC=<FILENAME> should build the executable in the subfolder

--- a/4_cv_basics/3_interpolation/README.md
+++ b/4_cv_basics/3_interpolation/README.md
@@ -127,15 +127,15 @@ This method is called "bilinear" because it uses linear interpolation in two dir
 ```
 2. Move into the Interpolation directory from the cv_basics directory by running following command :
 ```
-    cd 8_Interpolation 
+    cd 3_interpolation 
 ```
 3. Build the file by running the following command :
 ```
-    make SRC=main.cpp link=interpolation.cpp
+    make SRC=main.cpp link=src/interpolation.cpp
 ```
 4. Finally execute the binary file created by running the following command :
 ```
-    ./Interpolation
+    ./interpolation
 ```
 
 

